control-plane-operator/controllers/hostedcontrolplane/v2/cno/component.go

```diff
@@ -100,3 +102,65 @@ func SetRestartAnnotationAndPatch(ctx context.Context, crclient client.Client, d
 
 	return nil
 }
+
+type operand struct {
+	DeploymentName  string
+	ContainerName   string
+	ReleaseImageKey string
+}
+
+var (
+	operandsDeploymentsList = []operand{
+		{
+			DeploymentName:  "ovnkube-control-plane",
+			ContainerName:   "ovnkube-control-plane",
+			ReleaseImageKey: "ovn-kubernetes",
+		},
+		{
+			DeploymentName:  "network-node-identity",
+			ContainerName:   "approver",
+			ReleaseImageKey: "ovn-kubernetes",
+		},
+		{
+			DeploymentName:  "multus-admission-controller",
```

Comment:
fwiw this will need to account for multus optional https://github.com/openshift/hypershift/pull/6358

Reply:
should we wait until that PR is merged then

Reply:
fixed